server/app/services/ambassadors.js

```diff
@@ -306,46 +302,69 @@ async function sendTriplerCountsToHubspot(ambassador) {
   let query =
-    "MATCH (a:Ambassador {id:$a_id})-[:CLAIMS]->(t:Tripler) RETURN t.first_name as first_name, t.last_name as last_name, t.address as address, t.verification as verification"
+    "MATCH (a:Ambassador {id:$a_id})-[:CLAIMS]->(t:Tripler) WHERE  t.status <> 'unconfirmed' RETURN t.first_name as first_name, t.last_name as last_name, t.address as address, t.verification as verification"
   let collection = await neode.cypher(query, {a_id: ambassador.get("id")})
   let verificationStrings = []
   for (let index = 0; index < collection.records.length; index++) {
+    let invidualTriplerMatchScore = 0
+    let address = collection.records[index].get("address").toLowerCase()
     let verificationString = collection.records[index].get("verification").toString().toLowerCase()
     let triplerProperties = []
     triplerProperties.push(collection.records[index].get("first_name").toLowerCase())
     triplerProperties.push(collection.records[index].get("last_name").toLowerCase())
-
+    triplerProperties.push(address.match(re)[0])
     for (let i in triplerProperties) {
-      if (verificationString.includes(verificationString[i])) {
-        triplerSubScore += 1
-      } else {
-        triplerSubScore += -1
+      if (verificationString.includes(triplerProperties[i])) {
+        invidualTriplerMatchScore++
       }
     }
+
+    if (invidualTriplerMatchScore >= 1) {
+    // if the individual tripler has one or more matches (good), then the overall tripler blemish is made smaller
+      tripler_ekata_blemish = tripler_ekata_blemish - triplerEkataBonus
+    } else {
+    // if the individual tripler has less than one match (bad), then the overall tripler blemish is made larger
+      tripler_ekata_blemish = tripler_ekata_blemish + triplerEkataPenalty
+    }
   }
 
-  // calculate the Ambassador's SubScore
-  let verificationString = ambassador.get("first_name").toLowerCase()
+  // determining the blemishness of the ambassador
+  let verificationString = ambassador.get("verification").toLowerCase()
+  let address = ambassador.get("address").toLowerCase()
   let ambassadorProperties = [
     ambassador.get("first_name").toLowerCase(),
     ambassador.get("last_name").toLowerCase(),
+    address.match(re)[0]
   ]
   for (let i in ambassadorProperties) {
     if (verificationString.includes(ambassadorProperties[i].toLowerCase())) {
-      ambassadorSubScore += 1
+      ambassadorMatches++
     }
   }
 
-  ekataMatchScore = ambassadorSubScore + Math.max(0, triplerSubScore)
-  ambassador.update({ekata_match_score: ekataMatchScore})
+  // if the ambassador does not have enough matches, levy the penalty
+  if (ambassadorMatches < ambassadorEkataThreshold) {
+    ambassador_ekata_blemish = ambassadorEkataPenalty
+  }
+
+  ambassador.update({
+    ambassador_ekata_blemish: ambassador_ekata_blemish,
+    tripler_ekata_blemish: Math.max(0,tripler_ekata_blemish), //tripler_ekata_blemish should not be negative, floor is 0
```

Comment:
Can we let the `tripler_ekata_blemish` go negative in the return and take care of the `max` within `isLocked`? That way we have more visibility into the final fraud calculation.

Reply:
discussed in slack to not do this yet